package.json

```diff
@@ -71,6 +74,7 @@
     "matrix-js-sdk": "^26.1.0",
     "pg": "^8.11.2",
     "prom-client": "^14.2.0",
+    "react": "^18.2.0",
```

Comment:
Why was react added as a dependency?

Reply:
Our captcha widget renders with react. Our packages import `@prosopo/common` which has `"react-i18next` as a dependency. It's a package for doing translations in react. It depends on react. Unfortunately, when I install with `yarn`, react is not installed in the dependency tree. I had to manually add it here to force installation.

This was a while ago so I'll try removing this, removing node_modules and doing the installation again to confirm this is still the case.

Reply:
Have removed react as a dep